src/StripeTests/StripeMockFixture.cs

```diff
@@ -110,8 +110,8 @@ public string GetFixture(string path, string[] expansions = null)
         /// <returns>-1 if a &gt; b, 1 if a &lt; b, 0 if a == b.</returns>
         private static int CompareVersions(string a, string b)
         {
-            var version1 = new Version(a);
-            var version2 = new Version(b);
+            var version1 = new System.Version(a);
```

Comment:
Given the issue in the PR description, does this properly keep the -beta or does this convert it to .0?

Reply:
System.Version does not treat -beta properly. We don't need it since stripe-mock version does not have beta flag. 
I had to change this file was because of the addition of new Stripe.Version class. System.Version was being used earlier and this change just resolves ambiguity for the  compilers.

Reply:
Gotcha.  I would suggest leaving a comment in the code about this, but otherwise that makes sense.